frontend/src/views/components/FriendApproveCard.tsx

```diff
@@ -36,7 +35,7 @@ export const FriendApproveCard = ({
         console.log(data, userId, id, "aprovecard");
         if (data && data.length > 0) {
           // eslint-disable-next-line @typescript-eslint/no-unused-vars
-          const { data: updatedData, error: updateError } = await supabase
+          const { data: updatedData, error: updateError } = await client
             .from("Friendships")
             .update({ statusId: 2, updatedAt: new Date() })
```

Comment:
At this point, it's okay! However, as a next step, you should define constants of status. Because it's difficult to understand `statusId = 2` means what. Is this active, pending, or suspended?

Reply:
Will refactor at other issue.